package.json

```diff
@@ -77,6 +78,7 @@
     "gulp-shell": "^0.8.0",
     "gulp-sourcemaps": "^3.0.0",
     "gulp-terser": "^2.0.1",
+    "gulp-typescript": "^6.0.0-alpha.1",
```

Comment:
alpha...

![I also like to live dangerously](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExbG42ejJtNnBvcGMydjU3aWZjejg5cnF3bGlzYmxjYTFsdjV1ZzF3NSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/DwIdasRkFKsMg/giphy.gif)